grafana-plugin/src/models/filters/filters.ts

```diff
@@ -3,21 +3,39 @@ import { action, observable } from 'mobx';
 import BaseStore from 'models/base_store';
 import { makeRequest } from 'network';
 import { RootStore } from 'state';
+import { getItem, setItem } from 'utils/localStorage';
 
 import { getApiPathByPage } from './filters.helpers';
 import { FilterOption, FiltersValues } from './filters.types';
 
+const LOCAL_STORAGE_FILTERS_KEY = 'grafana.oncall.global-filters';
+
 export class FiltersStore extends BaseStore {
   @observable.shallow
   public options: { [page: string]: FilterOption[] } = {};
 
   @observable.shallow
   public values: { [page: string]: FiltersValues } = {};
 
-  public globalValues: FiltersValues = {};
+  private _globalValues: FiltersValues = {};
 
   constructor(rootStore: RootStore) {
     super(rootStore);
+
+    const savedFilters = getItem(LOCAL_STORAGE_FILTERS_KEY);
+    if (savedFilters) {
+      this.globalValues = { ...savedFilters };
+    }
```

Comment:
should we add an `else` clause here which sets to the user's default configured team if nothing is stored in local storage?

Reply:
good idea, but default team is unknown at the moment of this code running...